### PR TITLE
use EnumValue for GitHubActionTrigger

### LINKED ISSUE #409: GitHub Actions Trigger "pullrequest" is not a valid event name.
### Description
GitHubActionsTrigger not using EnumValue to generate yaml file.

### Steps to reproduce

1. Attribute in Build.cs:
```
[GitHubActions("Compile", 
  GitHubActionsImage.UbuntuLatest, 
  AutoGenerate = true, 
  On = new []{ GitHubActionsTrigger.PullRequest, GitHubActionsTrigger.Push },
  ImportGitHubTokenAs = nameof(GitHubToken),
  InvokedTargets = new []{ nameof(Compile) })]
```

### Relevant log output
Generates yaml:
```
name: Compile

on: [pullrequest, push]

jobs:
  ubuntu-latest:
    name: ubuntu-latest
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v1
      - name: Run './build.cmd Compile'
        run: ./build.cmd Compile
        env:
            GitHubToken: ${{ secrets.GITHUB_TOKEN }}
```

`on: [pullrequest, push]` should be `on: [pull_request, push] `

### PR DESCRIPTION
**Pull request by MikeBairdRocks**
fixes #409  
I confirm that the pull-request:

- [X] Follows the contribution guidelines
- [X] Is based on my own work
- [X] Is in compliance with my employer

### CODE REVIEW

File: source/Nuke.Common/CI/GitHubActions/Configuration/GitHubActionsConfiguration.cs [hunk 1]
```diff
@@ -1,9 +1,10 @@
-// Copyright 2019 Maintainers of NUKE.
+﻿// Copyright 2019 Maintainers of NUKE.
```
Comment (matkoch) [inline source/Nuke.Common/CI/GitHubActions/Configuration/GitHubActionsConfiguration.cs]: If you remove this change, I can merge the PR :)

Comment (MikeBairdRocks) [reply source/Nuke.Common/CI/GitHubActions/Configuration/GitHubActionsConfiguration.cs]: Weird, that diff does not show on my machine.

Comment (MikeBairdRocks) [reply source/Nuke.Common/CI/GitHubActions/Configuration/GitHubActionsConfiguration.cs]: Ok, GitKraken was set to CRLF 🙄

**Comment by MikeBairdRocks** [comment]
rebuild

**Comment by MikeBairdRocks** [comment]
Looks like the MacOS build had a transient error. "The HTTP request timed out after 00:01:40."

**Comment by matkoch** [comment]
Thanks for your contribution! 😄🎉

**Pull request merged**